java/src/main/java/com/google/oak/transport/GrpcStreamingTransport.java

```diff
@@ -16,11 +16,6 @@
 
 package com.google.oak.transport;
 
-import java.time.Duration;
```

Comment:
I don't think these formatting changes should be part of the PR.

Reply:
It happened automatically. Will revert.

Reply:
Done